core/src/main/resources/swagger/rawls.yaml

```diff
@@ -3308,6 +3308,21 @@ paths:
             - openid
             - email
             - profile
+  '/api/user/groups':
+    get:
+      responses:
+        '200':
+          description: Successful Request
+      description: "get the list of groups the user belong to"
```

Comment:
nit: don't need quotes; also, "belong" -> "belongs"

for completeness, specify the response structure, something like:
```
        '200':
          description: Successful Request
          schema:
            type: array
            description: list of the user's groups
            items:
              type: string
```